docs/concepts/data_lifecycle_policy.rst

```diff
@@ -0,0 +1,12 @@
+.. _data-lifecycle-policy:
+
+Data Lifecycle Policy
+*********************
+
+What it is
+##########
+
+Aspects is a data pipeline that captures, transforms, and aggregates tracking logs from the Open edX platform into xAPI statements and stores them in a ClickHouse database.
+However, the data is not stored indefinitely by default. The data is keep for 1 year by default, but this can be adjusted by the site operator via the setting `ASPECTS_DATA_TTL_EXPRESSION` in the tutor plugin.
+
+The setting value is a ClickHouse expression that defines the TTL policy for the data. The expression is evaluated for each row in the table and should return a date. Rows with a date in the past are deleted. You can read more about the TTL policy in the ClickHouse documentation: https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/mergetree/#ttl
```

Comment:
Can you describe what "TTL" means here, and also add a bit about how this works with our partitioning scheme? We should recommend that operators align their TTL with the end of the month so that CH can drop whole partitions instead of having to rewrite an entire months worth of data per table.